tests/e2e/package.json

```diff
@@ -37,6 +37,7 @@
     "edit-json-file": "1.7.0",
     "eslint": "7.32.0",
     "eslint-plugin-import": "2.24.2",
+    "sqlite3": "^5.0.10",
```

Comment:
Highly recommend pinning the version.
Where is the `package-lock`?

Reply:
Pinned the version. We use yarn.lock instead of package-lock, added this file also